scripts/src/lib/Template.svelte

```diff
@@ -242,6 +244,16 @@
             </p>
         </div>
 
+        <div>
+            <div class="option-container">
+                <input id="kotlinGradle" type="checkbox" class="option-input" bind:checked={useKotlinGradle} />
+                <label for="kotlinGradle" class="option-label">Gradle Kotlin DSL</label>
+            </div>
+            <p class="option-body">
+                Use the <a href="https://docs.gradle.org/current/userguide/kotlin_dsl.html">Kotlin DSL</a> for the Gradle build script. This option is only reccomended for advanced users familar with Kotlin and Gradle. This may result in slower build peformance and compability issues with some IDEs.
```

Comment:
```suggestion
                Use the <a href="https://docs.gradle.org/current/userguide/kotlin_dsl.html">Kotlin DSL</a> for the Gradle build script. This option is only recommended for advanced users familiar with Kotlin and Gradle. This may result in slower build performance and compatibility issues with some IDEs.
```

Reply:
oops..
Thank you for noticing this.